swarm/swarm.go

```diff
@@ -51,19 +57,29 @@ func run(cmd string) {
 }
 
 func getleaderHost() string {
```

Comment:
@smothiki cam you verify this change? (Is not "easy" to obtain the leader in etcd2)